src/pages/login/login.ts

```diff
@@ -14,11 +21,36 @@ import { NavController, NavParams } from 'ionic-angular';
 })
 export class LoginPage {
 
-  constructor(public navCtrl: NavController, public navParams: NavParams) {
+  private loginCreds : FormGroup;
+
+  constructor(public navCtrl: NavController, public navParams: NavParams, public _userService: UserProvider, private formBuilder: FormBuilder) {
```

Comment:
Coordinate with @digitalkyopo on naming conventions for _user/_userService. I think you and him named it different things.